criu/pie/restorer.c

```diff
@@ -1418,7 +1418,7 @@ long __export_restore_task(struct task_restore_args *args)
 	n_helpers = args->helpers_n;
 	zombies = args->zombies;
 	n_zombies = args->zombies_n;
-	*args->breakpoint = rst_sigreturn;
+	*args->breakpoint = ARCH_SIGRETURN_ADDR(rst_sigreturn);
```

Comment:
ARCH_SIGRETURN_ADDR isn't required. Breakpoints are used to stop a process as close as possible to the target system call. Then we call PTRACE_SYSCALL to stop a process on the syscall-exit point.